awx_collection/plugins/modules/tower_credential.py

```diff
@@ -274,6 +282,12 @@
       vault_password: 'new_password'
       vault_id: 'My ID'
 
+- name: Copy Credential
+  tower_credential:
+    name: Copy password
+    copy_from: Example password
+    credential_type: Vault
```

Comment:
We should modify these params so that credential_type is not required. It could be confusing if you are copying a machine credential and specify a vault type.

Reply:
Credentials requires Kind or Credential type atm, regardless of copy, and I believe we are phasing out kind, where credential type will then be required, This was more just not messing with the basic functioning of the module, and credential_type can be machine as it is on line 210 of the tower_credential/tasks/main.yml test.